django/db/migrations/serializer.py

```diff
@@ -155,11 +155,6 @@ def serialize(self):
             raise ValueError("Cannot serialize function: lambda")
         if self.value.__module__ is None:
             raise ValueError("Cannot serialize function %r: No module" % self.value)
-        # Python 3 is a lot easier, and only uses this branch if it's not local.
```

Comment:
Python 3 does indeed use this branch as `__qualname__` is a 3.3 and up feature, I believe. This code triggers for an object that is in a global scope (so it can give a full import path), and the code below it triggers for Python 2 and objects in a local scope. If anything could be removed, it's some of the code below, but I'm not sure which combination at the moment, as some of it is definitely used.

In short, though, don't remove this!